README.md

```diff
@@ -21,7 +21,9 @@ To deploy the cluster you can use :
 
 ```ShellSession
 # Install dependencies from ``requirements.txt``
-sudo pip3 install -r requirements.txt
+python3 -m venv ~/venv-kubespray
```

Comment:
Maybe we should just link to the ansible.md (https://kubespray.io/#/docs/ansible?id=installing-ansible) section instead of copying the venv install instructions across the docs.

Reply:
/hold cancel